sdk/core/ci.yml

```diff
@@ -1,10 +1,24 @@
 # DO NOT EDIT THIS FILE
 # This file is generated automatically and any changes will be lost.
 
+resources:
+  repositories:
+    - repository: azure-sdk-tools
+      type: github
+      name: Azure/azure-sdk-tools
+      endpoint: azure
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/heads/net-unified-pipelines
```

Comment:
Do we intend to keep this for the initial set go around?

Reply:
Just a question as I kind of like it but I'm curious what your thinking around it.

Reply:
I was going to switch back to master. But we should have a chat about using this mechanism to point to a tag of the build tools repo, perhaps as a step of mothballing the servicing branch.